src/Export/Base/XlsxRenderer.php

```diff
@@ -39,6 +39,18 @@ protected function saveSpreadsheet(Spreadsheet $spreadsheet): string
             throw new \Exception('Could not open temporary file');
         }
 
+        // Enable auto filter
+        $sheet = $spreadsheet->getActiveSheet();
+        $sheet->setAutoFilter('A1:' . $sheet->getHighestColumn() . '1');
+
+        // Freeze first row
+        $sheet->freezePane('B2');
```

Comment:
I donÄt get why that is B2, but its seems to work^^

Reply:
It's this weird cell logic of spreadsheet apps.

You want to lock something? It wants to know the upper left corner that is locked. Never understood it mayself 100%. I hope B2 is okay with all possible exports.